src/common/src/catalog/mod.rs

```diff
@@ -126,6 +126,57 @@ impl From<TableId> for u32 {
     }
 }
 
+#[derive(Clone, Copy, Debug, Display, Default, Hash, PartialOrd, PartialEq, Eq, Ord)]
+pub struct ColumnIdx {
```

Comment:
This struct seems irrelevant to the module `catalog`. 😢 Can we simply use the bare `usize` for the column index, which is widely used in other occurrences?

Reply:
Of cause we can :D I only created `ColumnIdx` to give future coders an easy understanding of what this represents. `usize` works just as well for me.